startupscript/butane/attach-docker-cache.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# attach-docker-cache.sh scans for any newly attached docker cache disks and
```

Comment:
When and where do we execute this script?

Reply:
This will be run when the system boots, before docker starts. Added to the comment

Reply:
Okay, and you plan to call this in a followup PR separate from https://github.com/verily-src/verily1/pull/54040/files?